Vectorizer naming update (#158)
timescale/pgai: docs/vectorizer.md

This page offers a comprehensive overview of Vectorizer features,
demonstrating how it streamlines the process of working with vector embeddings

> in your database. To quickly test drive embeddings using a pre-built Docker developer environment, see the 
> [Vectorizer quick start](/docs/vectorizer-quick-start.md). For a more detailed technical specification, see the

Reviewer: ```suggestion
in your database. To quickly try out embeddings using a pre-built Docker developer environment, see the 
[Vectorizer quick start](/docs/vectorizer-quick-start.md). For a more detailed technical specification, see the
```

- Self-hosted Postgres

  Set an environment variable that is the [same as your API key name](./vectorizer-worker.md#install-and-configure-vectorizer-worker). 
  For example:
  ```bash
  export OPENAI_API_KEY="Your OpenAI API key"

>   pgai vectrorizer worker

Reviewer: ```suggestion
```
What is this call of vectorizer for here? The `vectorizer.md` explains the SQL vectorizer interface. It has nothing to do with the worker and calling it here is not necessary and just confusing to me.

   chunk,

>    embedding <=> <query embedding> as distance

Reviewer: ```suggestion
   embedding <=> ai.openai_embed(...) as distance
```

Since we are now adding secret fetching to cloud we can do the openai embedding call here. This is much nicer because it allows copy pasting the docs and actually doing semantic search.

Can be a separate PR though to change this.
